src/main/java/org/cyclonedx/maven/BaseCycloneDxMojo.java

```diff
@@ -217,6 +218,12 @@ public abstract class BaseCycloneDxMojo extends AbstractMojo {
     @Parameter( defaultValue = "${project.build.outputTimestamp}" )
     private String outputTimestamp;
 
+    /**
+     * External references to be added to <code>$.metadata.component.externalReferences[]</code>.
+     */
```

Comment:
need @since

Reply:
Fixed in df528da6db0628634ac4313a97cd18701d2c3e71.